src/highdicom/sr/templates.py

```diff
@@ -2918,13 +2925,65 @@ def roi(self) -> Union[ImageRegion, ImageRegion3D, None]:
         if len(matches) > 1:
             logger.warning(
                 'found more than one "Image Region" content item '
-                'in "Planar ROI Measurements and Qualitative Evalutions" '
+                'in "Planar ROI Measurements and Qualitative Evaluations" '
                 'template'
             )
-        if len(matches) == 0:
+        if len(matches) > 0:
             return ImageRegion3D.from_dataset(matches[0])
         return None
 
+    @property
+    def referenced_segment(
+        self
+    ) -> Union[ImageContentItem, None]:
+        """Union[highdicom.sr.ImageContentItem, None]:
+        segmentation frame referenced by the measurements group
+        """  # noqa: E501
+        root_item = self[0]
+
+        # Find the referenced segmentation frame content item
+        matches = find_content_items(
+            root_item,
+            name=codes.DCM.ReferencedSegmentationFrame,
+            value_type=ValueTypeValues.IMAGE
+        )
+        if len(matches) > 1:
+            logger.warning(
+                'found more than one "Referenced Segmentation Frame" content '
+                'item in "Planar ROI Measurements and Qualitative Evaluations" '
+                'template'
+            )
+        elif len(matches) == 0:
+            return None
+
+        return ImageContentItem.from_dataset(matches[0])
```

Comment:
Should they already be of type `ImageContentItem`?

Reply:
Yes, I was copying some of the existing code, which does not assume that content items have been converted to their highdicom types. I guess it was written before the "eager" type conversion was implemented and could now be simplified.

Reply:
Resolved in bd0f5d3